js/components/ManuscriptForm.js

```diff
@@ -1,89 +1,121 @@
 // The list could come from a json object with name and date
-const ManuscriptForm = () => {
-  return (
-    <form className={"manuscript-form"}>
-      <div className={"field"}>
-        <label className={"control"}>Select Manuscripts:</label>
-        <div className={"control"}>
-          <div className={"select is-multiple"}>
-            <select multiple>
-              <option>Manuscript A</option>
-              <option>Manuscript B</option>
-              <option>Manuscript C</option>
-              <option>Manuscript D</option>
-              <option>Manuscript E</option>
-              <option>Manuscript F</option>
-            </select>
+
+class ManuscriptForm extends React.Component {
+  render() {
+    return (
+      <form className={"manuscript-form"}>
+        <div className={"field"}>
+          <label className={"control"}>Select Manuscripts:</label>
+          <div className={"control"}>
+            <div className={"select is-multiple"}>
+              <select multiple>
+                <option>Vat. Syr. 157</option>
+                <option>Vat. Syr. 161</option>
+                <option>Vat. Syr. 283</option>
+                <option>Vat. Syr. 586</option>
+                <option>Vat. Syr. 252</option>
+                <option>Bor. Syr. 13</option>
+                <option>BL. Add. 12144</option>
+                <option>BL. Add. 12139</option>
+                <option>BL. Add. 12146</option>
+              </select>
+            </div>
+          </div>
+          <div className={"control sort-option"}>
+            <p>Order manuscripts by:</p>
+            <label className={"radio"}>
+              Shelfmark:
+              <input type={"radio"} name={"Shelfmark"} defaultChecked />
+            </label>
+            <label className={"radio"}>
+              Date:
+              <input type={"radio"} name={"Date"} />
+            </label>
           </div>
         </div>
-        <div className={"control sort-option"}>
-          <p>Order manuscripts by:</p>
-          <label className={"radio"}>
-            Shelfmark:
-            <input type={"radio"} name={"Shelfmark"} defaultChecked />
-          </label>
-          <label className={"radio"}>
-            Date:
-            <input type={"radio"} name={"Date"} />
-          </label>
-        </div>
-      </div>
-      <div className={"field"}>
-        <label className={"control"}>Select Letters:</label>
-        <div className={"control"}>
-          <div className={"select is-multiple"}>
-            <select multiple>
-              <option>ʾĀlep̄</option>
-              <option>Bēṯ</option>
-              <option>Gāmal</option>
-              <option>Dālaṯ</option>
-              <option>Hē</option>
-              <option>Waw</option>
-            </select>
+        <div className={"field"}>
+          <label className={"control"}>Select Letters:</label>
+          <div className={"buttons are-small"}>
+            <LetterButton letter="ܐ" />
```

Comment:
This chunk would be better done having an array of letters, then using a map statement to generate the individual LetterButton components.

Reply:
Sure -- I've made the change via a new commit.